res/database/Default/user/scr0llbaer_constr_people.xml

```diff
@@ -0,0 +1,927 @@
+		<person id="837d8cdd-38de-4253-9a60-05cc17f900b7" first_name="Conny" last_name="Heynes" first_name_original="Connie" last_name_original="Hines" gender="2" birthday="1931-03-24" deathday="" imdb_id="nm0385647" creator="9551" />
+		<person id="49c74240-b643-40c7-9453-06e26cc4cdbc" first_name="Lenny" last_name="Keyten" first_name_original="Larry" last_name_original="Keating" gender="1" birthday="1899-06-13" deathday="1963-08-26" imdb_id="nm0444122" creator="9551" />
+		<person id="1d86a38d-4b31-4993-aa60-7bf8bd4396c5" first_name="Ben" last_name="Hencock" first_name_original="Bob" last_name_original="Hathcock" gender="1" birthday="1944-09-19" deathday="" imdb_id="nm0368906" creator="9551" />
+		<person id="657e1f85-bce8-46f6-87d7-b84b13de758a" first_name="Heinz Helmut" last_name="Dockeff" first_name_original="Hans Helmut" last_name_original="Dickow" gender="1" birthday="1927-04-14" deathday="1989-12-18" imdb_id="nm0225588" creator="9551" />
+		<person id="04203ce9-0b38-406b-9d4f-d8c54339cc70" first_name="Elke" last_name="Haardegen" first_name_original="Edith" last_name_original="Heerdegen" gender="2" birthday="1913-09-02" deathday="1982-07-13" imdb_id="nm0373492" creator="9551" />
+		<person id="0825b522-3856-43a2-a4e8-6344f49f2030" first_name="Dieter" last_name="Krabbs" first_name_original="Diether" last_name_original="Krebs" gender="1" birthday="1947-08-11" deathday="2000-01-04" imdb_id="nm0470522" creator="9551" />
+		<person id="ac8abee5-d4d1-4295-9ae8-57f9346b9864" first_name="Kelvin" last_name="Serbo" first_name_original="Kevin" last_name_original="Sorbo" gender="1" birthday="1958-09-24" deathday="" imdb_id="nm0001757" creator="9551" />
+		<person id="10f3e1ca-490e-45f2-aafb-885f22317701" first_name="Mitchell" last_name="Burst" first_name_original="Michael" last_name_original="Hurst" gender="1" birthday="1957-09-20" deathday="" imdb_id="nm0403628" creator="9551" />
+		<person id="5e4777f9-b18e-4d2a-89c1-035bddb9b105" first_name="Lucie" last_name="Losslaw" first_name_original="Lucy" last_name_original="Lawless" gender="2" birthday="1968-03-29" deathday="" imdb_id="nm0005128" creator="9551" />
+		<person id="90a27935-757b-4a36-a8c7-d145f03da366" first_name="Darell" last_name="Dyke" first_name_original="Daryl" last_name_original="Duke" gender="1" birthday="1929-03-08" deathday="" imdb_id="nm0241174" creator="9551" />
+		<person id="e8146c67-f105-4680-8ef7-c51d0aa2e258" first_name="Raquel" last_name="Weird" first_name_original="Rachel" last_name_original="Ward" gender="2" birthday="1957-09-12" deathday="" imdb_id="nm0001829" creator="9551" />
+
+		<!-- fictional -->
+
+		<person id="8565d7d6-7c8d-447f-acd7-5dc65f073bec" first_name="Penelope" last_name="Truthseeker" nick_name="Trutherpenny" gender="2" birthday="1931-09-11" deathday="2012-12-21" country="GB" job="137" bookable="1" fictional="1" creator="9551" />
+	
+	</insignificantpeople>
+
+	<exportOptions onlyFakes="true" onlyCustom="false" dataStructure="FakeData" />
```

Comment:
don't know where these options are really used, but onlyFakes is set incorrectly; may not matter when merging is done via moving entries to existing files

Reply:
about no. 17 in the OP? I think I tried to set it to 'true' but it resulted in the same behavior. I found no documentation for that option. It seems to mean it only has fictional entries, but I would have thought the separation of fictional from 'real' data is a rather new 'invention'.

Reply:
that entry is ... ancient. It was added by my export script when creating the database from the old mysql-data (the time of the online editor ...)

Reply:
https://github.com/TVTower/TVTower/issues/1064

Reply:
Cause of the problem is not this line. Just remove it.